target/linux/ath79/image/mikrotik.mk

```diff
@@ -9,6 +9,15 @@ define Device/mikrotik_routerboard-493g
 endef
 TARGET_DEVICES += mikrotik_routerboard-493g
 
+define Device/mikrotik_routerboard-911-5hn-nand
+  $(Device/mikrotik_nand)
+  SOC := ar9344
+  DEVICE_MODEL := RouterBOARD 911-5Hn NAND (911 Lite5)
+  DEVICE_PACKAGES += rssileds kmod-gpio-beeper
```

Comment:
There is no beeper on 911 board on both variants. Please remove package and move the dtsi node to appropriate dts.

Reply:
@adschm, @tmn505,

Thanks for the review.

> Alternatively, I'd suggest e.g. "-nand128M" and "-nor16M" suffixes,

> I wonder if image name shouldn't be changed to routerboard-911-xhn-nand, since 2Hn version will work with the same image.

I added the -nand128m suffix, and changed the name to mention the flash size. I also added the 2.4 GHz version 2Hn in `mikrotik.mk`:
```
define Device/mikrotik_routerboard-911-5hn-nand128m
  $(Device/mikrotik_nand)
  SOC := ar9344
  DEVICE_MODEL := RouterBOARD 911-5Hn NAND 128MB (911 Lite5)
  DEVICE_ALT0_VENDOR := MikroTik
  DEVICE_ALT0_MODEL := RouterBOARD 911-2Hn NAND 128MB (911 Lite2)
  DEVICE_PACKAGES += rssileds
  SUPPORTED_DEVICES += rb-911-5hn-nand
endef
TARGET_DEVICES += mikrotik_routerboard-911-5hn-nand128m
```

> There is no beeper on 911 board on both variants. Please remove package and move the dtsi node to appropriate dts.

I removed the beeper and moved the node to the SXT Lite 5 dts.

> Inevitably the NOR variant will be added, which will further clobber the visibility if kept like this. For readability full flash table would be preferable.
> As @f00b4r0 said, fixed means no size auto-detection guaranteed ever, so it could break in the future. For reference the flash map:

I created a whole SPI NOR partition table on each device.

> Additionally there is ethernet LED which works on ar71xx image but not with ath79.

Unfortunately, I can't verify this, since I don't own the device.

Reply:
Except for the Ethernet LED, which I can't test, I'd say everything else here is solved.